### PR TITLE
[GPU] Disable onednn globally when model has convolutions with weights zero points

### PR DESCRIPTION
**Pull request by jade-cho**
### Tickets:
 - *142818*

### CODE REVIEW

File: src/plugins/intel_gpu/src/graph/graph_optimizer/reorder_inputs.cpp [hunk 1]
```diff
@@ -59,7 +59,19 @@ std::map<program_node*, format::type> get_preferred_formats(program& p, layout_o
...
+        if (n->is_type<convolution>()) {
+            total_convs++;
+            if (n->as<convolution>().weights_zero_points_term())
+                num_wei_zp_convs++;
+        }
+    }
+
+    GPU_DEBUG_LOG << "Number of convolutions with weights zero points: " << num_wei_zp_convs << "/" << total_convs << std::endl;
+
+    if ((onednn_impls_counter < 1 && lo.get_optimization_attributes().use_onednn_impls) || (num_wei_zp_convs > 0)) {
```
Comment (isanghao) [inline src/plugins/intel_gpu/src/graph/graph_optimizer/reorder_inputs.cpp:74]: Proper condition will be:
((onednn_impls_counter < 1 || (num_wei_zp_convs > 0) && lo.get_optimization_attributes().use_onednn_impls)

Comment (jade-cho) [reply src/plugins/intel_gpu/src/graph/graph_optimizer/reorder_inputs.cpp:74]: How is it?
if (lo.get_optimization_attributes().use_onednn_impls && (onednn_impls_counter < 1 || num_asym_wei_convs > 0)) {

**Review by isanghao — APPROVED** [review]
LGTM. let me check the performance impact.

**Comment by isanghao** [comment]
No perf issue except for 1 outlier. It is acceptable.

**Pull request merged**